app/pages/sections/order-items/catalogue-solutions/order-item/flat/ondemand/ui-tests/general.ui.test.js

```diff
@@ -0,0 +1,347 @@
+    .expect(priceFieldWithError.exists).ok()
+    .expect(await extractInnerText(priceField.find('#price-error'))).contains(content.errorMessages.PriceRequired);
+});
+
+test('should anchor to the price field when clicking on the price required error link in errorSummary ', async (t) => {
+  await pageSetup({ ...defaultPageSetup, postRoute: true });
+  await t.navigateTo(pageUrl);
+
+  const saveButton = Selector('[data-test-id="save-button"] button');
+  const errorSummary = Selector('[data-test-id="error-summary"]');
+  const priceInput = Selector('[data-test-id="question-price"] Input');
+
+  await t
+    .expect(errorSummary.exists).notOk()
+    .selectText(priceInput).pressKey('delete')
+    .click(saveButton);
+
+  await t
+    .click(errorSummary.find('li a').nth(0))
+    .expect(getLocation()).eql(`${pageUrl}#price`);
+});
+
+test('should anchor to the price field when clicking on the numerical price error link in errorSummary ', async (t) => {
+  await pageSetup({ ...defaultPageSetup, postRoute: true });
+  await t.navigateTo(pageUrl);
+
+  const saveButton = Selector('[data-test-id="save-button"] button');
+  const errorSummary = Selector('[data-test-id="error-summary"]');
+  const price = Selector('[data-test-id="question-price"]');
+
+  await t
+    .expect(errorSummary.exists).notOk()
+    .typeText(price, 'blah', { paste: false })
+    .click(saveButton);
+
+  await t
+    .click(errorSummary.find('li a').nth(0))
+    .expect(getLocation()).eql(`${pageUrl}#price`);
+});
+
+test('should anchor to the price field when clicking on the price required error link in errorSummary ', async (t) => {
+  await pageSetup({ ...defaultPageSetup, postRoute: true });
+  await t.navigateTo(pageUrl);
+
+  const saveButton = Selector('[data-test-id="save-button"] button');
+  const errorSummary = Selector('[data-test-id="error-summary"]');
+  const priceInput = Selector('[data-test-id="question-price"] input');
+
+  await t
+    .expect(errorSummary.exists).notOk()
+    .selectText(priceInput).pressKey('delete')
+    .click(saveButton);
+
+  await t
+    .click(errorSummary.find('li a').nth(0))
+    .expect(getLocation()).eql(`${pageUrl}#price`);
+});
+
+test('should anchor to the price field when clicking on the numerical price error link in errorSummary ', async (t) => {
+  await pageSetup({ ...defaultPageSetup, postRoute: true });
+  await t.navigateTo(pageUrl);
+
+  const saveButton = Selector('[data-test-id="save-button"] button');
+  const errorSummary = Selector('[data-test-id="error-summary"]');
+  const priceInput = Selector('[data-test-id="question-price"] input');
+
+  await t
+    .expect(errorSummary.exists).notOk()
+    .typeText(priceInput, 'blah', { paste: true })
+    .click(saveButton);
+
+  await t
+    .click(errorSummary.find('li a').nth(0))
+    .expect(getLocation()).eql(`${pageUrl}#price`);
+});
+
+test('should render solution table as errors with error message when no practice list sizes are entered', async (t) => {
```

Comment:
Similar to above, `practice list sizes` should be `quantities`.

Reply:
Done